PresentScreenings/ControlsFactory.cs

```diff
@@ -57,6 +61,16 @@ public static NSButton CreateCancelButton(CGRect frame)
             return cancelButton;
         }
 
+        public static NSComboBox NewRatingComboBox(CGRect frame, NSFont font)
```

Comment:
`NewSomething` is a better name than `CreateSomething` because the method returns something.
Well done!

Please consider changing the names of the `CreateSomething` methods.

Reply:
Done.